devnets/Config.ts

```diff
@@ -25,11 +25,13 @@ export interface DevChainConfig {
 
   chain: string
   nodes?: number
+  genesis?: Record<string, unknown>
```

Comment:
I don't think this should be configurable in the config unless we have a really solid use-case (that can't be solved by something else)